frontend/src/scenes/dashboard/SharedDashboard.scss

```diff
@@ -0,0 +1,32 @@
+@import '~/vars';
+
+.SharedDashboard {
+    min-height: 100vh;
+    padding: 1rem;
+
+    &.SharedDashboard--embedded {
+        padding: 0.5rem 1rem;
+    }
+
+    .SharedDashboard-header {
+        .SharedDashboard-header-team {
+            display: none;
+        }
+
+        @media screen and (min-width: $md) {
```

Comment:
We should try to avoid sass vars as much as possible. Is there a css var you can use instead?

Reply:
It isn't. Would rather leave this for a time where we methodically tackle it globally